### DISCUSSION TITLE
littleFS install problem

### DISCUSSION THREAD

**Discussion by hakha4** [fredlcore/BSB-LAN#569] [Q&A] [answered]
Hi all!
I've been using BSB-lan ver 2 for some time and it works very well (apart from loosing wifi and needs to be restarted) but I want to upgrade to latest version. Installing on a ESP32 wroom in Arduino IDE ver 2.04. When compiling littleFS.h is not found and I can't find it to install from library manager. There is a file littlefs (Alexander Entinger), downloaed this but still not working,

What littleFS file is expected ? Where to download ?
Any help appreciated
Regards Håkan
BTW awesome work of developers !

**Comment by fredlcore** [comment, marked as answer]
Thanks!
You shouldn't have to download any library because LittleFS is part of the ESP32 core since framework version 2.x. I assume you don't use the most recent version of the core (possibly just 1.0.6 or so), so if you use the link to the ESP32 framework mentioned in our manual, it should compile without any errors.

**Comment by hakha4** [comment]
Thank's for quick response!  Updating ESP32 core solved problem. Thank's
BTW are circuit boards still avaliable for purchase ?

**Comment by fredlcore** [comment]
Good to hear :). And yes, boards are readily available, send me an e-mail to bsb (ät) code-it.de
